test-network/README.md

```diff
@@ -3,3 +3,22 @@
 You can use the `./network.sh` script to stand up a simple Fabric test network. The test network has two peer organizations with one peer each and a single node raft ordering service. You can also use the `./network.sh` script to create channels and deploy chaincode. For more information, see [Using the Fabric test network](https://hyperledger-fabric.readthedocs.io/en/latest/test_network.html). The test network is being introduced in Fabric v2.0 as the long term replacement for the `first-network` sample.
 
 Before you can deploy the test network, you need to follow the instructions to [Install the Samples, Binaries and Docker Images](https://hyperledger-fabric.readthedocs.io/en/latest/install.html) in the Hyperledger Fabric documentation.
+
+## Using the Peer commands
```

Comment:
I think this is a good enhancement.
However, we previously decided to put all test network doc in the Fabric ReadTheDocs tutorial, rather than having instructions spanning both the README and tutorial, so that we can have a single place for readers and for our own maintenance sanity. Do you want to move the README content to the test network docs tutorial?

Reply:
@denyeart I'll put in a separate PR for the fabric docs and update this  to refer back. when it's avilable.